tools/cc-policy/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Please delete this file - unless you think we need it. We have a top-level LICENSE file in this repo.